index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tribute Page</title>
+
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+
+    <style>
+      #image {
+        max-width: 100%;
+      }
+
+      img {
+        display: block;
+      }
+    </style>
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img id="image" />
+        <div id="img-caption">Here is Image</div>
+      </div>
+      <p id="tribute-info">Here is tribute info</p>
+      <a
+        id="tribute-link"
+        href="https://github.com/Rana-Mudassir/microverse_Mudassir-Kwabena.git"
+        target="_blank"
+      >
```

Comment:
Linters errors should be resolved. Closing tags should be used

Reply:
Please remember to include why you suggest a change. Some things might be obvious to you but not to someone else. :+1: :smiley: :smiley:

Reply:
- [ ] Please consider editing this comment too, as Santiago suggested! Always ask yourself: How applying the requested changes can improve the code base? In this particular case, How does resolving the linter errors and using closing tags can improve the code base?

Please, also consider being more polite when rephrasing your comments, they should sound like suggestions and not orders! 👍🏼 

💡 Hint:

> * This is a polite suggestion: *"I think it would be better to use semantic tags"*
> * This reads as an order: *"Use semantic tags"*

Please, apply these changes! 🤓 🛠️